src/cli.js

```diff
@@ -454,9 +465,10 @@ function plural(count) {
         setSigint(); // attach SIGINT (Ctrl + C) handler after runner.init(), so user can Ctrl + C out of a long branchify operation via the default SIGINT handler
 
         // Create smashtest directory if it doesn't already exist
-        const SMASHTEST_DIR = 'smashtest';
+        const SMASHTEST_DIR = reporter.getPathFolder();
```

Comment:
Since these vars are being set dynamically now, I wonder if we should make them lowercase, like all other vars. What do you think?

Reply:
Yeah we can use Camel Case, i change it